components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -1023,6 +1023,14 @@ private void setThrottleProperties(OMElement element) {
                 throttleProperties.setEnabledSubscriptionLevelSpikeArrest(JavaUtils.isTrueExplicitly
                         (enabledSubscriptionLevelSpikeArrestElement
                                 .getText()));
+            }
+            // Check Immediate SubscriptionPolicyUpdate
+            OMElement enableImmediateSubscriptionPolicyUpdateElement =
+                    throttleConfigurationElement.getFirstChildWithName(new QName(APIConstants.AdvancedThrottleConstants
+                            .ENABLE_IMMEDIATE_SUBSCRIPTION_POLICY_UPDATE));
+            if (enableImmediateSubscriptionPolicyUpdateElement != null) {
+                throttleProperties.setEnableImmediateSubscriptionUpdate(JavaUtils.isTrueExplicitly
```

Comment:
Can't we simply use Boolean.parseBoolean()?

Reply:
Front porting changes from 3.2.0.